adapters/audienceNetwork/facebook_test.go

```diff
@@ -40,3 +42,21 @@ type FacebookExt struct {
 func TestJsonSamples(t *testing.T) {
 	adapterstest.RunJSONBidderTest(t, "audienceNetworktest", NewFacebookBidder(nil, "test-platform-id", "test-app-secret"))
 }
+
+func TestMakeTimeoutNotice(t *testing.T) {
+	req := adapters.RequestData{
+		Body: []byte(`{"imp":[{"id":"1234"}]}}`),
```

Comment:
Could we also add another test case for an invalid json in this test?

Reply:
Done